Configuration/Skimming/python/Skims_PbPb_cff.py

```diff
@@ -50,3 +50,16 @@
     )
 
 #####################      
+
+from Configuration.Skimming.PbPb_ZMuSkimMuonDPG_cff import *
+ZMuSkimPath = cms.Path( diMuonSelSeq )
+SKIMStreamZMu = cms.FilteredStream(
+    responsible = 'HI PAG',
+    name = 'ZMu',
```

Comment:
@mandrenguyen for naming consistency with the other skims I would pre-pend a PbPb to ZMu